rust_src/src/fns.rs

```diff
@@ -0,0 +1,43 @@
+extern crate libc;
```

Comment:
Sure. It put in fns.rs just to keep parity with fns.c, however I think it does make more sense to be in the 'strings' module.